client/src/constants/gtm-tags.js

```diff
@@ -31,4 +31,9 @@ module.exports = {
     medical: "_MH",
     other: "_OH",
   },
+  step1: {
```

Comment:
I think it's better to group the tags under `offerHelp` property. It's more descriptive.

Reply:
ill change it to offer help instead